CMakeLists.txt

```diff
@@ -14,7 +14,8 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
   "${CMAKE_CURRENT_LIST_DIR}/cmake/CeleritasMakeRulesOverride.cmake")
 
 project(Celeritas VERSION "${Celeritas_VERSION}" LANGUAGES CXX)
-cmake_policy(VERSION 3.18...3.28)
+cmake_policy(VERSION 3.18...3.31)
+cmake_policy(SET CMP0175 OLD)
```

Comment:
Please revert this line; instead, we should fix whatever is broken in the CMake policy. https://cmake.org/cmake/help/latest/policy/CMP0175.html#cmp0175